src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -1,39 +1,36 @@
 package edu.kis.powp.jobs2d.command.gui;
 
-import java.awt.*;
+import java.awt.Container;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
 import java.awt.event.ActionEvent;
 import java.util.List;
 
-import javax.swing.*;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JTextArea;
 
-import java.util.ArrayList;
-import edu.kis.legacy.drawer.panel.DrawPanelController;
-import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.appbase.gui.WindowComponent;
-import edu.kis.powp.jobs2d.Job2dDriver;
-import edu.kis.powp.jobs2d.command.DriverCommand;
-import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
```

Comment:
Why there are commits changing CommandManagerWindow? It seems you make a mess! You should embrace changes introduced in other PR-s, not delete them. Look carefully at your PR and fix the code you broke. 
If it would be difficult, fork the main repo again and start from scratch.